### PR TITLE
Have TestLogger print out duration instead of time

### PR DESCRIPTION
**Pull request by akalin-keybase**
Printing out the time on every log consumes a lot of
columns, and printing out a duration seems more informative.

### CODE REVIEW

**Comment by akalin-keybase** [comment]
Example output:

```
--- PASS: TestBlockJournalBasic (0.00s)
block_journal.go:531: 8.483µs [D] Putting 4 bytes of data for block 019f64a747e1b97f131fabb6b447296c9b6f0201e79fb3c5356e6c77e89b6a806a with context BlockContext{Creator: 01000000000000000000000000000000}
block_journal.go:613: 1.428999ms [D] Adding reference for block 019f64a747e1b97f131fabb6b447296c9b6f0201e79fb3c5356e6c77e89b6a806a with context BlockContext{Creator: 01000000000000000000000000000000, Writer: 02000000000000000000000000000000, RefNonce: a87657da8b822125}
block_journal.go:261: 1.927767ms [D] Reading journal entries 0 to 1
block_journal.go:352: 2.039953ms [D] Found 4 block bytes in the journal
block_journal.go:261: 2.08035ms [D] Reading journal entries 0 to 1
block_journal.go:352: 2.129948ms [D] Found 4 block bytes in the journal
block_journal.go:261: 2.235959ms [D] Reading journal entries 0 to 1
block_journal.go:352: 2.290467ms [D] Found 4 block bytes in the journal
PASS
```

File: go/logger/test_logger.go [hunk 1]
```diff
@@ -31,103 +32,115 @@ type TestLogBackend interface {
...
 }
 
 // Verify TestLogger fully implements the Logger interface.
 var _ Logger = (*TestLogger)(nil)
 
-func prefixCaller(extraDepth int, lvl logging.Level, fmts string) string {
+func (log *TestLogger) prefixCaller(
+	extraDepth int, lvl logging.Level, fmts string) string {
+	log.setStartTimeOnce.Do(func() {
+		*log.startTime = time.Now()
```
Comment (akalin-keybase) [inline go/logger/test_logger.go]: Done.

**Comment by akalin-keybase** [comment]
I implemented the suggestion to print out the start time on first log, but that revealed a subtle problem. If there are multiple TestLoggers per test, then the durations will depend on which TestLogger it's based on.

Filed https://github.com/keybase/kbfs/pull/536 to start cleaning this up.

**Comment by akalin-keybase** [comment]
Also, re. your concerns about testability, it seems to me that most tests benefit more from readable logs than explicit timestamp info. For the few that do, they can print out timestamps as needed, or maybe having the start time plus the various durations is enough.

But I'll ping #core to make sure they're okay with it.

**Comment by jzila** [comment]
What's the status of this PR?

**Comment by akalin-keybase** [comment]
Eh, I haven't gotten back to it, and it's not that important. I'll just close.

**Pull request closed without merging**